src/components/DataCollector/index.js

```diff
@@ -14,18 +16,32 @@ const createDataCollector = ({ eventManager }) => {
   return {
     commands: {
       event(options) {
+        let { xdm } = options;
         const {
-          xdm,
           data,
           viewStart = false,
-          documentUnloading = false
+          documentUnloading = false,
+          eventType,
+          eventMergeId
```

Comment:
Did you consider calling these `type` and `mergeId` instead of `eventType` and `eventMergeId` since we're already in an "event" command? I'm not sure which is better...be concise or be consistent with XDM names?

Reply:
I was just following the JIRA ticket.  @jfkhoury or @dancingcactus what do you think?

Reply:
I like concise a lot more, but the reason we are exposing them here is to be consistent with XDM. It’ll require a quick addition in the doc to mention that those Params are mapped to the XDM values. 

I’ll leave it up to @dancingcactus.

Reply:
Love the more concise language.

Reply:
Is this what mobile will be doing as well? I want to be consistent.

Reply:
Talked to Pete, he doesn't think consistency is an issue.